feathr_project/feathr/__init__.py

```diff
@@ -13,3 +13,4 @@
 from .lookup_feature import LookupFeature
 from .aggregation import Aggregation
 from .feathr_configurations import SparkExecutionConfiguration
+from feathr.api.app.core.feathr_api_exception import *
```

Comment:
I made the call that the exceptions in [feathr.api.app.core.fathr_api_exception.py](https://github.com/linkedin/feathr/blob/main/feathr_project/feathr/api/app/core/feathr_api_exception.py) should be exported by default as part of the public API. Happy to roll this back if reviewers feel otherwise.